### PR TITLE
Fix run of SupportService command

### PR DESCRIPTION
**Pull request by bonewell**

### CODE REVIEW

**Comment by VladSemenyuk** [comment]
Reviewed

**Comment by KKarlash** [comment]
Reviewed

File: src/components/vr_module/include/vr_module/vr_proxy.h [hunk 1]
```diff
@@ -68,6 +68,7 @@ class VRProxy : public threads::MessageLoopThread<MessageQueue>::Handler {
   inline void StartChannel();
   void Receive();
   void Handle(vr_hmi_api::ServiceMessage message);
+  void OnEstablished();
```
Comment (DKlimenko) [inline src/components/vr_module/include/vr_module/vr_proxy.h]: please provide description for each function

Comment (DKlimenko) [inline src/components/vr_module/include/vr_module/vr_proxy.h]: on Established - confusing name

**Comment by DKlimenko** [comment]
reviewed

File: src/components/vr_module/include/vr_module/vr_proxy.h (hunk 1 above)
Comment (bonewell) [inline src/components/vr_module/include/vr_module/vr_proxy.h]: What name can you suggest?

**Pull request merged**